README.md

```diff
@@ -414,18 +414,32 @@ services:
 | Parameter                                | Description                                                                   | Default        |
 |:---------------------------------------- |:----------------------------------------------------------------------------- |:-------------- |
 | `alerting.pagerduty`                     | Configuration for alerts of type `pagerduty`                                  | `{}`           |
-| `alerting.pagerduty.integration-key`     | PagerDuty Events API v2 integration key.                                      | Required `""`  |
+| `alerting.pagerduty.integration-key`     | PagerDuty Events API v2 integration key.                                      | `""`           |
 | `alerting.pagerduty.default-alert`       | Default alert configuration. <br />See [Setting a default alert](#setting-a-default-alert) | N/A       |
+| `alerting.pagerduty.integrations`        | Pagerduty integrations per team configurations                                |                |
+| `alerting.pagerduty.integrations`        | Pagerduty integrations per team configurations                                |   `[]`         |
+| `alerting.pagerduty.integrations`        | Pagerduty integrations per team configurations                                |   `[]`         |
+| `alerting.pagerduty.integrations[].integration-key` | Pagerduty integrationkey for a perticular team                     |   `""`         |
+| `alerting.pagerduty.integrations[].group`           | the group that the integration key belongs to                      |   `""`         |
 
 It is highly recommended to set `services[].alerts[].send-on-resolved` to `true` for alerts
 of type `pagerduty`, because unlike other alerts, the operation resulting from setting said
 parameter to `true` will not create another incident, but mark the incident as resolved on
 PagerDuty instead.
 
+Behavior:
+- Team integration have priority over the general integration
+- If no team integration is provided it will defaults to the general pagerduty integration 
+- If no team integration and no general integration were provided it defaults to the first team integration provided
+
+
 ```yaml
 alerting:
   pagerduty: 
     integration-key: "********************************"
+    intergrations:
+     - integration-key: "********************************"
+       group: "core"
```

Comment:
So if I understand correctly, `alerting.pagerduty.integration-key` is the fallback integration key and the `alerting.pagerduty.integrations[]` with a matching group name takes precedence over the fallback integration key, right?

Reply:
Yes Correct.